clients/unshaded/src/main/java/tachyon/client/file/policy/RoundRobinPolicy.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.client.file.policy;
+
+import java.util.List;
+
+import tachyon.client.WorkerNetAddress;
+import tachyon.client.block.BlockWorkerInfo;
+
+/**
+ * A policy that chooses the worker for the next block in a round-robin manner and skips workers
+ * that do not have enough space. The policy returns null if no worker can be found.
+ */
+public final class RoundRobinPolicy implements FileWriteLocationPolicy {
+  private List<BlockWorkerInfo> mWorkerInfoList;
+  private int mIndex;
```

Comment:
Will keeping state in this class work? Since this class will only be per FileOutStream? Won't the distributions of blocks be skewed if index is always initialized as 0? (Given that generally number of blocks per file is less than number of total workers)